### ISSUE TITLE
TOOL: Clean up PROD forms tab

### ISSUE THREAD

**Issue by HasanSifat** [bcgov/digital-journeys#674]
The PROD environment forms tab has multiple entries of the telework form. Need them cleaned out. Iman had previously looked into this issue but did not get solved yet. 

Task

- [ ] Confirm users can see only one instance of the telwork form to submit new

- [ ] Confirm there is only one edition of SL Review form too

**Comment by MeghanStothers** [comment]
Iman noted the best time would be after the upgrade - should take an hour or two,

@simensma-fresh  Any dependencies on the database?

**Comment by MeghanStothers** [comment]
@HasanSifat  to make Fazil a new user to see what he can see

**Comment by iman-jamali-fw** [comment]
My investigation shows that the issue that causes extra entries of a form is no longer an issue on v5 as I tested this on my local and TEST environments. Neither uploading a new version of a form on top of an existing one nor deleting a form and uploading a newer version of that form causes extra form entries.

However, there are 9 Telework entries on PROD that were side-effects of a bug in the form-process association (wizard that associates a workflow to a form) which was fixed right after discovery. 

Looking into different options on how to remove those extra entries on PROD and will update this ticket.
